src/main/java/com/altenheim/kalender/controller/logicController/IOController.java

```diff
@@ -123,14 +88,50 @@ public void loadContactsFromFile() throws IOException, ClassNotFoundException
     {
         var path = settings.getPathToIcsExportedFile();
         if (path == null)
-            path = "contactFiles/contacts.file"; 
+            path = "contactFiles/contacts.file";
         var loadFile = new FileInputStream(path);
         var inputStream = new ObjectInputStream(loadFile);
         var loadedContacts = (List<ContactModel>)inputStream.readObject();
         allContacts.addAll(loadedContacts);
         inputStream.close();
-    }  
-    
+    }
+
```

Comment:
Die ganze Klasse hat noch Redundanzen, würde sagen wenn alles drin ist was das Laden von serialisierten Objekten angeht können wir das refactorn, dann haben wir nen guten Überblick welche Bestandteile wir auslagern können in eigene Methoden.